decisions/47-use-spdx-licenses.md

```diff
@@ -0,0 +1,87 @@
+# Use SPDX license vocabulary for URIs
+
+Discussion: https://github.com/ESIPFed/science-on-schema.org/issues/47
+
+## Status ##
+_Proposed_
+
+## Decision ##
+
+Use SPDX license URIs to unambiguously specify the license for data and metadata use.
+
+## Context ##
+
+Link a Dataset to its license to document legal constraints by adding a [schema:license](https://schema.org/license) property. The [guide](https://developers.google.com/search/docs/data-types/dataset) recommends providing a URL that unambiguously identifies a specific version of the license used, but for many licenses it is hard to determine what that URL should be. Thus, we recommend that the license URL be drawn from the [SPDX license list](https://spdx.org/licenses/), which provides a curated list of licenses and their properties that is well maintained. For each SPDX entry, SPDX provides a canonical URL for the license (e.g., `http://spdx.org/licenses/CC0-1.0`), a unique `licenseId` (e.g., `CC0-1.0`), and other metadata about the license. Here's an example using the SPDX license URI for the Creative Commons CC-0 license:
+
+<pre>
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+  },
+  "@id": "http://www.sample-data-repository.org/dataset/123",
+  "@type": "Dataset",
+  "name": "Removal of organic carbon by natural bacterioplankton communities as a function of pCO2 from laboratory experiments between 2012 and 2016",
+  <strong>"license": "http://spdx.org/licenses/CC0-1.0"</strong>
+  ...
+}
+</pre>
+
+While many licenses are ambiguous about the license URI for the license, the Creative Commons licenses are an exception in that they provide extremely consistent URIs for each license, and these are in widespread use.  So, while we recommend using the SPDX URI, we recognize that some sites may want to use the CC license URIs directly, which is acceptable.  Here's an example using the traditional CC URI for the license.
```

Comment:
... they provide ~~extremely~~ consistent URIs ...

~~So, while we recommend using the SPDX URI, we recognize that some sites may want to use the CC license URIs directly, which is acceptable.~~ **While we recommend using the SPDX URI, it is acceptable to use the CC license URIs directly if preferred.**

Reply:
Resolved in sha af17b3d.

Reply:
should the license be a resource?:
```
"license": { "@id": "http://spdx.org/licenses/CC0-1.0" }
```